src/discussions/tours/NotRespondedFilter.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { ProductTour } from '@edx/paragon';
+
+import { notRespondedFilterTour } from './data/selectors';
+import { fetchDiscussionTours, updateTourShowStatus } from './data/thunks';
+
+export default () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchDiscussionTours());
+  }, []);
+  const tourData = useSelector(notRespondedFilterTour);
+  const config = {
+    tourId: 'notRespondedTour',
+    advanceButtonText: 'Next',
+    dismissButtonText: 'Dismiss',
+    endButtonText: 'Okay',
+    enabled: tourData ? tourData.showTour : false,
```

Comment:
**Breaking Change**
This check will not work in the case of an empty object and the **code will break**. You are returning an empty object through your selector if the required data does not found. 
`{} ? true : false => output: true`

This check will work when the selector would return the data object or undefine/null.